test/test_helper.rb

```diff
@@ -77,4 +82,5 @@ def self.prepare_pv(name)
   end
 
   TestProvisioner.prepare_pv("pv0001")
+  TestProvisioner.prepare_pv("pv0002")
```

Comment:
If I understand correctly how this kind of PV works (as opposed to the dynamic kind you get via storage classes), the entire PV is reserved by PVCs made against it. So in theory pv0001 could still be getting recycled from a previous test/namespace's claim the next time we need it. I didn't actually see this happen though.